spaceship/lib/spaceship/tunes/build_train.rb

```diff
@@ -98,7 +123,7 @@ def update_testing_status!(new_value, testing_type, build = nil)
 
             # also update the builds
             train['builds'].delete_if do |b|
-              return true if b[testing_key].nil?
+              next true if b[testing_key].nil?
```

Comment:
✅  Yup. The return would actually return from the entire enclosing method, which was probably not what its original author intended. This is now part of the muti-part if statement.